scripts/lunch.coffee

```diff
@@ -0,0 +1,66 @@
+# Description:
+#   Lunch recording technology.
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   <botname> lunch at [location]
+#   <botname> lunch me
+
+makeRecord = (now = new Date()) ->
+
+  return {
+    last_lunch: now.getFullYear() + "-" + (now.getMonth() + 1) + "-" + now.getDate()
```

Comment:
I think that storing the last timestamp twice (once as a formatted string) smells a bit funny. This makes more sense as just presentation logic, doesn't really need to be cached...